test/fixtures/sample_playbooks.json

```diff
@@ -0,0 +1,52 @@
+{
```

Comment:
Is this format of fixtures given by something? It is weird to have numeric keys, given we are not using that format anywhere else.
I'd prefer to have the format that is given by the foreman_proxy here (which I think is an array of playbooks, right?

Reply:
sorry for that, forgot to change it on this branch :D 
I uploaded the new changes, hope it is OK now.